man/nvim.1

```diff
@@ -336,8 +336,22 @@ Print version information and exit.
 .El
 .Sh ENVIRONMENT
 .Bl -tag -width Fl
+.It Ev XDG_CONFIG_HOME
+Path to use for the user-local runtime directory, see
+.Sx FILES .
+Defaults to
+.Pa ~/.config
```

Comment:
@justinmk Because XDG spec has nothing to do with windows, it is a spec for \*nix systems *only*.